core/imageroot/usr/local/bin/acl-load

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (C) 2021 Nethesis S.r.l.
+# http://www.nethesis.it - nethserver@nethesis.it
+#
+# This script is part of NethServer.
+#
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see COPYING.
+#
+
+#
+# Load ACLs from cluster/acls key
+# This script must be executed on the worker server
+#
+
+import os
+import sys
+import agent
+
+os.environ.update(agent.read_envfile('/var/lib/nethserver/cluster/state/agent.env'))
+
+rdb = agent.redis_connect(privileged=True)
+
+role = rdb.execute_command('ROLE')[0];
+
+if role == 'slave':
+    for acl in rdb.smembers('cluster/acls'):
+        user = acl.split(" ",2)[1]
+        # replace existing user
+        rdb.acl_deluser(user)
+        rdb.execute_command(f'ACL set{acl}')
+
+    rdb.acl_save()
+    print("ACLs loaded")
+else:
+    print("This command should run only on worker node", file=sys.stderr)
+    sys.exit(1)
```

Comment:
```suggestion
    print("This is a leader node, nothing to do", file=sys.stderr)
    sys.exit(0)
```

Reply:
I prefer to have a clear statement: the command failed.

Is like calling a program without required parameters: it exists with non-zero code